acceptance-tests.md

```diff
@@ -0,0 +1,159 @@
+These acceptance tests are based on the [NERC Operational Use Cases](https://docs.google.com/document/u/0/d/1dHHpWvJD4b1mdXTDQhgWhiGeOLY7N7FmwvgI5VB3tP0/edit).
+
+Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-started/user-onboarding-on-NERC/](https://nerc-project.github.io/nerc-docs/get-started/user-onboarding-on-NERC/)
+
+
+## Managing Users
+
+
+
+1. **Request a new account**
+    * _As a new user, I should be able to create an account for myself in the NERC._
```

Comment:
What was the reasoning to use `*` instead of `1.` here?

Reply:
Ah, my thought was that it's not a step in the acceptance tests, so I didn't want to imply that it contained an action to be executed.